config/jetpack-cloud-stage.json

```diff
@@ -40,6 +40,7 @@
 		"jetpack/agency-dashboard": true,
 		"jetpack/backup-messaging-i3": true,
 		"jetpack/backup-retention-settings": true,
+		"jetpack/golden-ticket": false,
```

Comment:
plan to open it up to staging when ready for testing?

Reply:
Yes, I'll enable it on staging when it's at the point there could be a call for testing (if we were to do that).